frontends/web/src/components/forms/input.test.tsx

```diff
@@ -1,6 +1,6 @@
 /**
  * Copyright 2018 Shift Devices AG
- * Copyright 2021 Shift Crypto AG
+ * Copyright 2023 Shift Crypto AG
```

Comment:
generally one does not delete old copyrights. just extend using "2021, 2023" (or 2021-2023 if 2022 also had changes)

Reply:
Oups really? I don't recall we ever do that in bitbox-wallet-app

I'd prefer 2021-2023 as it makes more sense to me than `"2021, 2022, 2023, 2024` (I've never seen that anywhere)

Reply:
I checked all Copyright headers, nowhere we use multiple years.. I'll just undo the copyright changes in this PR.